widget/editor.go

```diff
@@ -643,7 +643,9 @@ func (e *Editor) layout(gtx layout.Context, textMaterial, selectMaterial op.Call
 	key.InputOp{Tag: &e.eventKey, Hint: e.InputHint, Keys: keys}.Add(gtx.Ops)
 	if e.requestFocus {
```

Comment:
Wouldn't it be better if `e.requestFocus` is not set when `e.ReadOnly`? Then read-only Editors will never reach this branch, and not ask for focus with the `key.FocusOp` below.